src/index.ts

```diff
@@ -220,6 +218,8 @@ async function appBoot() {
 			} );
 		} );
 
+		await setupWPServerFiles().catch( Sentry.captureException );
```

Comment:
I suppose there is a (likely very small) possibility that the `activate` event fires before this promise resolves, as that event handler is adjacent to this scope. That would mean a user could still encounter the targeted error. 

Seems like a very unlikely scenario that we do not need to address, but I share it nonetheless. 

https://github.com/Automattic/studio/blob/4aa60ff084ccc8fe7f1b305c883c18affafc8548/src/index.ts#L251-L257

Reply:
Yeah, I think that would probably be an edge case. I tested adding a delay when executing the `ready` event, and triggering the `ready` event by cliking on the dock icon of the Studio app results in an exception. This could be easily addressed by adding a condition in `activate` event to only create the main window when all the logic performed in `ready` is complete. I'll do this in a different PR.

**Error:**
```
Error occurred in handler for 'getAppGlobals': Error: No handler registered for 'getAppGlobals'
    at WebContents.<anonymous> (node:electron/js2c/browser_init:2:78277)
    at WebContents.emit (node:events:514:28)
    at WebContents.emit (node:domain:488:12)
```

Reply:
I came to realize this might be more common than we expect based on the issue Automattic/dotcom-forge#8016. I'll try to tackle it asap.